Source/FolioReaderPage.swift

```diff
@@ -478,7 +482,10 @@ extension UIWebView {
     }
     
     public override func canPerformAction(action: Selector, withSender sender: AnyObject?) -> Bool {
-        
+		guard readerConfig.useReaderMenuController else {
+			return false
```

Comment:
I have made the `FolioReaderWebView` class public to be able to override its `canPerformAction()` method.